diff --git a/contracts/burn-redeem-escrow/smartpy/burn-redeem-escrow-smartpy.py b/contracts/burn-redeem-escrow/smartpy/burn-redeem-escrow-smartpy.py
index 7de13f2..bf178f4 100644
--- a/contracts/burn-redeem-escrow/smartpy/burn-redeem-escrow-smartpy.py
+++ b/contracts/burn-redeem-escrow/smartpy/burn-redeem-escrow-smartpy.py
@@ -268,7 +268,7 @@ class BurnRedeemEscrow(sp.Contract):
 
         # Emit event with the updated paused state.
         sp.emit(
-            sp.record(paused=paused_state),
+            sp.record(paused=self.data.paused),
             tag=EVENT_TYPE_PAUSE_STATE_TOGGLED
         )
 
