diff --git a/exchange/js/exchange.js b/exchange/js/exchange.js
index d3873b3..3af15b8 100644
--- a/exchange/js/exchange.js
+++ b/exchange/js/exchange.js
@@ -795,7 +795,8 @@ async function pollForNFTUpdate(address, tradedTokens, timeout = 30000, interval
     timeout,
     interval,
     logPrefix: DEBUG ? '[exchange] ' : undefined,
-    fetchBeforePolling: true
+    fetchBeforePolling: true,
+    refetchOnTimeout: true
   });
 }
 
